benchmarks/src/main/kotlin/chat/UserWithoutFriends.kt

```diff
@@ -0,0 +1,40 @@
+package chat
+
+import kotlinx.coroutines.channels.Channel
+
+class UserWithoutFriends(id: Long,
+                         activity: Double,
+                         messageChannel: Channel<Message>,
```

Comment:
It would be better to use capacity as a parameter

Reply:
What do you mean by capacity?